README.md

```diff
@@ -1,4 +1,4 @@
-> You might also like [Caret](http://caret.io?ref=parsedown) - our Markdown editor for Mac / Windows / Linux.
+> You might also like [Caret](http://caret.io) - our Markdown editor for Mac / Windows / Linux.
```

Comment:
Instead of removing the `ref` parameter, could you instead add the `/` that should be there. Also, since there's a redirect in place anyway, we might as well link to the https version too :)

i.e. instead change it to `https://caret.io/?ref=parsedown`